### ISSUE TITLE
Terraform crashes on plan after upgrading NewRelic provider from 1.20 to 2.7

### ISSUE THREAD

**Issue by hawaiian-shirt-guy** [newrelic/terraform-provider-newrelic#882]
After upgrading the NewRelic provider for Terraform from 1.20 to 2.7 terraform crashes when running a Terraform plan. Importantly, I have two different environments, one of which runs both plans and applies fine, the other always crashes on plan. crash.log is attached.

Terraform version information:
$terraform --version
Terraform v0.12.21
+ provider.newrelic v2.7.0
+ provider.template v2.1.2
[crash.log](https://github.com/newrelic/terraform-provider-newrelic/files/5228459/crash.log)

**Comment by zlesnr** [comment]
```
2020/09/11 15:21:17 [TRACE] GRPCProvider: ReadResource
2020-09-11T15:21:17.540-0400 [DEBUG] plugin.terraform-provider-newrelic_v2.7.0: 2020/09/11 15:21:17 [INFO] Reading New Relic alert condition 1023709:15866792
2020-09-11T15:21:17.544-0400 [DEBUG] plugin.terraform-provider-newrelic_v2.7.0: panic: runtime error: index out of range [1] with length 1
2020-09-11T15:21:17.544-0400 [DEBUG] plugin.terraform-provider-newrelic_v2.7.0:
2020-09-11T15:21:17.544-0400 [DEBUG] plugin.terraform-provider-newrelic_v2.7.0: goroutine 10743 [running]:
2020-09-11T15:21:17.544-0400 [DEBUG] plugin.terraform-provider-newrelic_v2.7.0: github.com/newrelic/terraform-provider-newrelic/v2/newrelic.flattenNrqlTerms(0xc0004440
00, 0x2, 0x4, 0xc000a3a000, 0x1, 0x1, 0x0, 0xc0005bc5a0, 0x0)
2020-09-11T15:21:17.544-0400 [DEBUG] plugin.terraform-provider-newrelic_v2.7.0:         /home/runner/work/terraform-provider-newrelic/terraform-provider-newrelic/newre
lic/structures_newrelic_nrql_alert_condition.go:534 +0x809
2020-09-11T15:21:17.544-0400 [DEBUG] plugin.terraform-provider-newrelic_v2.7.0: github.com/newrelic/terraform-provider-newrelic/v2/newrelic.flattenNrqlAlertCondition(0
x16af10, 0xc0004376c0, 0xc0001d89a0, 0x8, 0xc0004376c0)
2020-09-11T15:21:17.544-0400 [DEBUG] plugin.terraform-provider-newrelic_v2.7.0:         /home/runner/work/terraform-provider-newrelic/terraform-provider-newrelic/newre
lic/structures_newrelic_nrql_alert_condition.go:408 +0x544
2020-09-11T15:21:17.544-0400 [DEBUG] plugin.terraform-provider-newrelic_v2.7.0: github.com/newrelic/terraform-provider-newrelic/v2/newrelic.resourceNewRelicNrqlAlertCo
nditionRead(0xc0001d89a0, 0x1c14940, 0xc0007b3b30, 0xc0001d89a0, 0x0)
2020-09-11T15:21:17.544-0400 [DEBUG] plugin.terraform-provider-newrelic_v2.7.0:         /home/runner/work/terraform-provider-newrelic/terraform-provider-newrelic/newrelic/resource_newrelic_nrql_alert_condition.go:407 +0x2c6
2020-09-11T15:21:17.544-0400 [DEBUG] plugin.terraform-provider-newrelic_v2.7.0: github.com/hashicorp/terraform-plugin-sdk/helper/schema.(*Resource).RefreshWithoutUpgrade(0xc00012be60, 0xc0003ba370, 0x1c14940, 0xc0007b3b30, 0xc00023c460, 0x0, 0x0)
2020-09-11T15:21:17.544-0400 [DEBUG] plugin.terraform-provider-newrelic_v2.7.0:         /home/runner/go/pkg/mod/github.com/hashicorp/terraform-plugin-sdk@v1.15.0/helper/schema/resource.go:460 +0x119
2020-09-11T15:21:17.544-0400 [DEBUG] plugin.terraform-provider-newrelic_v2.7.0: github.com/hashicorp/terraform-plugin-sdk/internal/helper/plugin.(*GRPCProviderServer).ReadResource(0xc00000f1f0, 0x2089120, 0xc000358180, 0xc000a00180, 0xc00000f1f0, 0xc000358180, 0xc0005c5ba0)
2020-09-11T15:21:17.544-0400 [DEBUG] plugin.terraform-provider-newrelic_v2.7.0:         /home/runner/go/pkg/mod/github.com/hashicorp/terraform-plugin-sdk@v1.15.0/internal/helper/plugin/grpc_provider.go:525 +0x3d8
2020-09-11T15:21:17.545-0400 [DEBUG] plugin.terraform-provider-newrelic_v2.7.0: github.com/hashicorp/terraform-plugin-sdk/internal/tfplugin5._Provider_ReadResource_Handler(0x1d60c80, 0xc00000f1f0, 0x2089120, 0xc000358180, 0xc000a00120, 0x0, 0x2089120, 0xc000358180, 0xc000b98380, 0x305)
2020-09-11T15:21:17.545-0400 [DEBUG] plugin.terraform-provider-newrelic_v2.7.0:         /home/runner/go/pkg/mod/github.com/hashicorp/terraform-plugin-sdk@v1.15.0/internal/tfplugin5/tfplugin5.pb.go:3269 +0x217
2020-09-11T15:21:17.545-0400 [DEBUG] plugin.terraform-provider-newrelic_v2.7.0: google.golang.org/grpc.(*Server).processUnaryRPC(0xc00010e000, 0x2095980, 0xc00027f200, 0xc00027c100, 0xc00051db90, 0x2898a90, 0x0, 0x0, 0x0)
2020-09-11T15:21:17.545-0400 [DEBUG] plugin.terraform-provider-newrelic_v2.7.0:         /home/runner/go/pkg/mod/google.golang.org/grpc@v1.29.1/server.go:1082 +0x50a
2020-09-11T15:21:17.545-0400 [DEBUG] plugin.terraform-provider-newrelic_v2.7.0: google.golang.org/grpc.(*Server).handleStream(0xc00010e000, 0x2095980, 0xc00027f200, 0xc00027c100, 0x0)
2020-09-11T15:21:17.545-0400 [DEBUG] plugin.terraform-provider-newrelic_v2.7.0:         /home/runner/go/pkg/mod/google.golang.org/grpc@v1.29.1/server.go:1405 +0xccd
2020-09-11T15:21:17.545-0400 [DEBUG] plugin.terraform-provider-newrelic_v2.7.0: google.golang.org/grpc.(*Server).serveStreams.func1.1(0xc000036410, 0xc00010e000, 0x2095980, 0xc00027f200, 0xc00027c100)
2020-09-11T15:21:17.545-0400 [DEBUG] plugin.terraform-provider-newrelic_v2.7.0:         /home/runner/go/pkg/mod/google.golang.org/grpc@v1.29.1/server.go:746 +0xa1
2020-09-11T15:21:17.545-0400 [DEBUG] plugin.terraform-provider-newrelic_v2.7.0: created by google.golang.org/grpc.(*Server).serveStreams.func1
2020-09-11T15:21:17.545-0400 [DEBUG] plugin.terraform-provider-newrelic_v2.7.0:         /home/runner/go/pkg/mod/google.golang.org/grpc@v1.29.1/server.go:744 +0xa1
2020-09-11T15:21:17.548-0400 [DEBUG] plugin: plugin process exited: path=/Users/johnlarison/work/ced-nr-configuration/.terraform/plugins/darwin_amd64/terraform-provider-newrelic_v2.7.0 pid=24861 error="exit status 2"
```

**Comment by zlesnr** [comment]
Thank you for the report here.

**Comment by zlesnr** [comment]
@awaiian-shirt-guy are any changes being made to the HCL to create this failure?

**Comment by hawaiian-shirt-guy** [comment]
@zlesnr I made some minor changes to my .tf files and variables as part of the upgrade from 1.20 to 2.7 to fix the errors and warnings that popped up as part of the syntax change, but that was it.

**Comment by zlesnr** [comment]
This should be resolved in `v2.7.3`. Let us know if any other issues come up.
